### PR TITLE
Code Cleaning :- Removing warnings tagged as Errors #19

### PR DESCRIPTION
**Pull request by abhishek58g**
Code Cleaning :- Removing warnings tagged as Errors #19

### CODE REVIEW

File: tests/TestApp/UniversalTestApp/UniversalTestApp.Windows/MainPage.xaml.cs [hunk 1]
```diff
@@ -50,38 +50,33 @@ namespace UniversalTestApp
...
             this.AccessToken.Text = string.Empty;
             string token = null;//await tokenBroker.GetTokenWithUsernamePasswordAsync();
             this.AccessToken.Text = token;
         }
 
-        private async void Button_Click_2(object sender, RoutedEventArgs e)
+        private void Button_Click_2(object sender, RoutedEventArgs e)
         {
             this.AccessToken.Text = string.Empty;
-            string token = await tokenBroker.GetTokenInteractiveWithMsAppAsync(new PlatformParameters(false));
```
Comment (kpanwar) [inline tests/TestApp/UniversalTestApp/UniversalTestApp.Windows/MainPage.xaml.cs]: why is this code removed?

File: tests/TestApp/UniversalTestApp/UniversalTestApp.Windows/MainPage.xaml.cs [hunk 2]
```diff
@@ -50,38 +50,33 @@ namespace UniversalTestApp
...
-            this.AccessToken.Text = token;
         }
 
-        private async void Button_Click_3(object sender, RoutedEventArgs e)
+        private void Button_Click_3(object sender, RoutedEventArgs e)
         {
             this.AccessToken.Text = string.Empty;
-            string token = await tokenBroker.GetTokenWithClientCredentialAsync();
```
Comment (kpanwar) [inline tests/TestApp/UniversalTestApp/UniversalTestApp.Windows/MainPage.xaml.cs]: why is this code removed?

File: tests/TestApp/UniversalTestApp/UniversalTestApp.Windows/MainPage.xaml.cs (hunk 1 above)
Comment (abhishek58g) [inline tests/TestApp/UniversalTestApp/UniversalTestApp.Windows/MainPage.xaml.cs]: The tokenBroker was never assigned any value in this class.Hence its default value was persisting which was null.This was giving an error and hence was removed.

File: tests/TestApp/UniversalTestApp/UniversalTestApp.Windows/MainPage.xaml.cs (hunk 2 above)
Comment (abhishek58g) [inline tests/TestApp/UniversalTestApp/UniversalTestApp.Windows/MainPage.xaml.cs]: The tokenBroker was never assigned any value in this class.Hence its default value was persisting which was null.This was giving an error and hence was removed.

File: src/MSAL.PCL.WinRT/MSAL.PCL.WinRT.csproj [hunk 3]
```diff
@@ -90,6 +92,9 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <CodeAnalysisRuleSet>Sdl7.0.ruleset</CodeAnalysisRuleSet>
```
Comment (kpanwar) [inline src/MSAL.PCL.WinRT/MSAL.PCL.WinRT.csproj]: add here <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>

File: src/MSAL.PCL.WinRT/MSAL.PCL.WinRT.csproj [hunk 4]
```diff
@@ -101,6 +106,9 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
+    <CodeAnalysisRuleSet>Sdl7.0.ruleset</CodeAnalysisRuleSet>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (kpanwar) [inline src/MSAL.PCL.WinRT/MSAL.PCL.WinRT.csproj]: add here <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>

File: src/MSAL.PCL.WinRT/MSAL.PCL.WinRT.csproj (hunk 3 above)
Comment (abhishek58g) [inline src/MSAL.PCL.WinRT/MSAL.PCL.WinRT.csproj]: added

File: src/MSAL.PCL.WinRT/MSAL.PCL.WinRT.csproj (hunk 4 above)
Comment (abhishek58g) [inline src/MSAL.PCL.WinRT/MSAL.PCL.WinRT.csproj]: added

**Pull request merged**